### DISCUSSION TITLE
Glances IP not detected

### DISCUSSION THREAD

**Discussion by Scales82** [htpcBeginner/docker-traefik#104] [Support]
Hey guys,
I cannot for the life of me get Glances to work right now. Traefik log is as follows:

`time="2020-11-21T05:49:38Z" level=error msg="service \"glances-svc\" error: unable to find the IP address for the container \"/glances\": the server is ignored" providerName=docker container=glances-docker-`

I've had this problem for a while, I think traefik 1 also did same thing. Any ideas? am i missing something stupid?

**Comment by dphildebrandt** [comment]
Hard to know unless you post your docker compose file. Just compare it line for line against the one in the repo.

**Reply by Scales82** [reply]
It is line for line

**Comment by SimpleHomelab** [comment]
is your glances running? Sometimes i see that in traefik logs when there is some problem with glances and it is not running.

**Reply by Scales82** [reply]
Sorted it out. It was running previously but for some reason since I removed it from my compose file its begun working again. I notice it is slightly different than before, could have been a new version. Cheers for replying anyway
